_commands/basics/df.md

```diff
@@ -1,24 +1,99 @@
----
```

Comment:
These lines with dashes on them need to remain in the file for it to render properly.

Follow the `README.md` in the root of the `c4cs` repo to see how to set the site up so you can view the changes you made.

I will review the rest of this when those changes are made so this file renders properly when it is viewed.